### PR TITLE
CRUD support Event and Task

### LINKED ISSUE #58: CRUD support Event and Task
- [x] Edit does not retain tags of the activity.
- [x] Complete does not support Event.
- [x] Complete does not support Activity with null fields.
- [x] C and U currently allows Activities with duplicate titles.
- [x] Task with status 'complete' does not load

### PR DESCRIPTION
**Pull request by kazuhiroinaba**
Fix #58, #60
Ready for review

### CODE REVIEW

File: src/main/java/seedu/taskman/logic/commands/CompleteCommand.java [hunk 1]
```diff
@@ -39,53 +40,62 @@ public CompleteCommand(int targetIndex) {
...
-	                    activityToComplete.getFrequency().get());
-	        	try {
-					task.setStatus(new Status(STATUS_COMPLETE));
-				} catch (IllegalValueException e) {
-					// TODO Auto-generated catch block
-					e.printStackTrace();
-				}
-	            afterComplete = new Activity(task);
-	        }
+		try {
```
Comment (AlexFJW) [inline src/main/java/seedu/taskman/logic/commands/CompleteCommand.java]: indentation

File: src/main/java/seedu/taskman/logic/commands/CompleteCommand.java [hunk 2]
```diff
@@ -39,53 +40,62 @@ public CompleteCommand(int targetIndex) {
...
+            initMembers();
+        } catch (IllegalValueException e) {
+            return new CommandResult(e.getMessage());
         }
         		
         try {
             model.deleteActivity(activityToComplete);
             model.addActivity(afterComplete);
             return new CommandResult(String.format(MESSAGE_SUCCESS, afterComplete.getTitle().title));
         } catch (UniqueActivityList.ActivityNotFoundException pnfe) {
```
Comment (alexfjw) [inline src/main/java/seedu/taskman/logic/commands/CompleteCommand.java]: Rename personNotFoundException (pnfe) to something more readable.

File: src/main/java/seedu/taskman/logic/commands/CompleteCommand.java [hunk 3]
```diff
@@ -39,53 +40,62 @@ public CompleteCommand(int targetIndex) {
...
-
             indicateAttemptToExecuteIncorrectCommand();
             return new CommandResult(Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
-
         } catch (UniqueActivityList.DuplicateActivityException e) {
-
             try {
```
Comment (alexfjw) [inline src/main/java/seedu/taskman/logic/commands/CompleteCommand.java:57]: Not part of your code, but do we really need to try to add the activity again, when the adding has failed previously (and resulted in the current exception?)

Seems like we can make do without this nested try block.

File: src/main/java/seedu/taskman/model/event/UniqueActivityList.java [hunk 4]
```diff
@@ -39,13 +39,28 @@ protected DuplicateActivityException() {
      */
     public UniqueActivityList() {}
 
+    //TODO Do we need this? Or is simply checking for equivalence of title sufficient?
     /**
-     * Returns true if the list contains an equivalent task as the given argument.
+     * Returns true if the list contains an equivalent activity as the given argument.
+     * Here, equivalence refers to equivalence of all fields in activity.
```
Comment (alexfjw) [inline src/main/java/seedu/taskman/model/event/UniqueActivityList.java:45]: Indeed. We should remove this method.

**Pull request merged**